hazelcast/src/test/java/com/hazelcast/test/starter/ConfigConstructor.java

```diff
@@ -184,4 +185,17 @@ public static Object getValue(Object obj, String getter)
         Method method = obj.getClass().getMethod(getter, null);
         return method.invoke(obj, null);
     }
+
+    private static Properties copy(Properties original) {
+        if (original == null) {
+            return null;
+        }
+
+        Properties copy = new Properties();
```

Comment:
Since Properties extends HashTable, you should be able to clone like this:

```

        Properties original = new Properties();
        Properties clone = new Properties();
        clone.putAll(original);
```